sqlalchemy_redshift/dialect.py

```diff
@@ -26,6 +26,7 @@
     pass
 else:
     from alembic.ddl.base import RenameTable
+
```

Comment:
Can you revert whitespace changes or refactor them to a separate PR?

Reply:
I can, but for some reason this particular whitespace was causing travis build to fail